header/verify.go

```diff
@@ -6,10 +6,25 @@ import (
 	"time"
 
 	"github.com/tendermint/tendermint/light"
+	"github.com/tendermint/tendermint/types"
 
 	libhead "github.com/celestiaorg/go-header"
 )
 
+// clockDrift defines how much new header's time can drift into
+// the future relative to the now time during verification.
+var clockDrift = 10 * time.Second
+
+// trustingPeriod is period through which we can trust a header's validators set.
+//
+// Should be significantly less than the unbonding period (e.g. unbonding
+// period = 3 weeks, trusting period = 2 weeks).
+//
+// More specifically, trusting period + time needed to check headers + time
+// needed to report and punish misbehavior should be less than the unbonding
+// period.
+var trustingPeriod = 168 * time.Hour
```

Comment:
**TODO** this will likely mess with something if there's a discrepancy in syncer.Params and this -- Need to figure out a better way to reconcile.

Reply:
Are we sure the syncer will still work when sycning blocks outside of the weak subjectivity period (with a trusted hash as the sync target)?
cc @liamsi also